ladybug/datatype/base.py

```diff
@@ -212,6 +265,74 @@ def _clean(self, unit):
                     ' ', '').replace(
                         '%', 'pct')
 
+    @property
+    def name(self):
+        """The data type name."""
+        if self._name is None:
+            return re.sub(r"(?<=\w)([A-Z])", r" \1", self.__class__.__name__)
+        else:
+            return self._name
+
+    @property
+    def units(self):
+        """A list of all acceptabledata type units."""
+        return self._units
+
+    @property
+    def si_units(self):
+        """A list of acceptable si_units."""
+        return self._si_units
+
+    @property
+    def ip_units(self):
+        """A list of acceptable ip_units."""
+        return self._ip_units
+
+    @property
+    def min(self):
+        """The minimum value of the data type."""
+        return self._min
+
+    @property
+    def max(self):
+        """The maximum possible value of the data type."""
+        return self._max
+
+    @property
+    def abbreviation(self):
+        """The abbreviation of the data type."""
+        return self._abbreviation
+
+    @property
+    def unit_descr(self):
+        """A description of the data type."""
```

Comment:
Is it a description of the unit as the name suggests or data type as the docstring suggests? If it is the second one then I suggest we should just call it description.

Reply:
No. It's a description of what the units mean.  I forgot to finish the doc string but I have finished it now.